src/main/java/com/verisign/storm/metrics/GraphiteMetricsConsumer.java

```diff
@@ -18,15 +18,16 @@
 import backtype.storm.task.IErrorReporter;
 import backtype.storm.task.TopologyContext;
 import com.google.common.base.Throwables;
-import com.verisign.storm.metrics.graphite.GraphiteAdapter;
-import com.verisign.storm.metrics.graphite.GraphiteCodec;
-import com.verisign.storm.metrics.graphite.GraphiteConnectionAttemptFailure;
+import com.verisign.storm.metrics.graphite.ConnectionFailureException;
+import com.verisign.storm.metrics.reporters.AbstractReporter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
-import java.net.InetSocketAddress;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
 import java.util.Collection;
+import java.util.HashMap;
 import java.util.Map;
 
 /**
```

Comment:
We also need to update the javadoc for this class, now that we support two backends.